src/pages/individual/Individual.jsx

```diff
@@ -58,6 +58,36 @@ export default function Individual() {
     [queryClient, id],
   );
 
+  const individualDataForFeaturedPhoto = useMemo(
+    () => {
+      const allAssets = reduce(
+        individualData?.encounters,
+        (memo, encounter) => {
+          const newAssets = map(
+            get(encounter, 'annotations', []),
+            annotation => ({
+              src: annotation?.asset_src,
+              guid: annotation?.asset_guid,
+              filename: annotation?.created
```

Comment:
Happy to no longer call this filename in favor of something like altText if that is desired.

Reply:
Definitely, thanks.

Reply:
Done